index.js

```diff
@@ -113,12 +113,13 @@ module.exports = {
     yoda: 'warn',
 
     // https://typescript-eslint.io/rules/
-    '@typescript-eslint/no-require-imports': 'error',
     '@typescript-eslint/default-param-last': 'warn',
     '@typescript-eslint/max-params': ['warn', { max: 4 }],
     '@typescript-eslint/method-signature-style': 'warn',
+    '@typescript-eslint/no-require-imports': 'error',
     '@typescript-eslint/no-shadow': 'warn',
     '@typescript-eslint/no-unused-expressions': 'warn',
+    '@typescript-eslint/no-useless-template-literals': 'warn',
```

Comment:
Could you move this from `ADDITIONAL RULES` section to `CUSTOMIZED RULES FROM EXTENDED CONFIGS`, please? :pray: 

Using the sections helps to keep an overview of what's being added beyond recommended vs what's tweaking a rule that's already in recommended (e.g. changing level or options).

Reply:
Done, thanks for the info :D